composer.json

```diff
@@ -28,7 +28,7 @@
         "bin-dir": "bin"
     },
     "support": {
-        "irc": "irc://irc.freenode.org/forkcms",
+        "slack": "https://fork-cms.herokuapp.com",
```

Comment:
but it is not among the options listed in the documentation:
    email: Email address for support.
    issues: URL to the issue tracker.
    forum: URL to the forum.
    wiki: URL to the wiki.
    irc: IRC channel for support, as irc://server/channel.
    source: URL to browse or download the sources.
    docs: URL to the documentation.
    rss: URL to the RSS feed.

what would be a better pick in that case ?